paddle/fluid/framework/ir/mkldnn/conv_bias_mkldnn_fuse_pass.cc

```diff
@@ -298,6 +298,9 @@ void ConvBiasFusePass::ApplyImpl(ir::Graph* graph) const {
       for (auto& attr : conv->Op()->GetAttrMap()) {
         desc.SetAttr(attr.first, attr.second);
       }
+      for (auto& attr : conv->Op()->GetRuntimeAttrMap()) {
+        desc.SetAttr(attr.first, attr.second);
+      }
```

Comment:
Why is it needed only in `conv_bias_mkldnn_fuse_pass` and not in other conv fusions?

Reply:
The attributes setting in conv_bias_mkldnn_fuse_pass is special.
In other conv fusions, the extra attributes were set by attr name directly, for example:

<img width="611" alt="image" src="https://user-images.githubusercontent.com/13048366/187154372-92868666-73b6-4203-b33b-96090a2fde30.png">

<img width="609" alt="image" src="https://user-images.githubusercontent.com/13048366/187154520-0cb3b436-84e2-4362-9e23-f3c2d097b034.png">